src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,44 @@
+package lotto
+
+import kotlin.math.floor
+
+class Lotto {
+    private val _numbers = mutableListOf<Int>()
+    val numbers: List<Int> get() = _numbers
+
+    fun autoLotto(): Set<Int> {
+        return (NUMBER_MINIMUM..NUMBER_MAXIMUM).shuffled().take(LOTTO_NUMBER).toSortedSet()
+    }
+
+    fun generate(numbers: Set<Int>) {
+        require(numbers.size == 6) {
+
+        }
+        _numbers.addAll(numbers)
+    }
+
+    fun getPrize(userNumber: List<Int>, winningNumber: List<Int>): List<Rank> {
+        val counts = Ticket().purchasedLotto.map { lotto ->
+            lotto.numbers.count { _number -> winningNumber.contains(_number) }
+        }
+        return counts.map { count -> Rank.findMatchCount(count) }
+    }
```

Comment:
Lotto 한게임의 경우 하나의 Rank만 반환되어야 할 것 같아요.

추가로 userNumber 파라미터는 사용하지 않는 것 같네요, 제거가 필요해 보입니다.

```suggestion
fun getPrize(winningNumber: List<Int>): Rank {
    return Rank.findMatchCount(this._numbers.count { winningNumber.contains(it) })
}
```